test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -229,7 +229,7 @@ def _run_dependency_check_test(image, ec2_connection, processor):
     html_output = ec2_connection.run(f"cat ~/{dependency_check_report}", hide=True).stdout
     cves = re.findall(r">(CVE-\d+-\d+)</a>", html_output)
     vulnerabilities = set(cves) - allowed_vulnerabilities
-
+    severity = None
```

Comment:
Please define this inside the loop, right before the start of the `try` block. Also, please set it to `"UNKNOWN"`.